src/main/java/org/minions/devfund/noemiguzman/battleship/Ocean.java

```diff
@@ -0,0 +1,201 @@
+package org.minions.devfund.noemiguzman.battleship;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    private static final int SIZE_OCEAN = 20;
+    private static final int NRO_SHIPS = 13;
+    private Ship[][] ships = new Ship[SIZE_OCEAN][SIZE_OCEAN];
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+
+    /**
+     * Creates an empty ocean (fills the ships array with a bunch of EmptySea instances).
+     * Also initializes any game variables, such as how many shots have been fired.
+     */
+    public Ocean() {
+        shotsFired = 0;
+        hitCount = 0;
+        shipsSunk = 0;
+        for (int i = 0; i < SIZE_OCEAN; i++) {
+            for (int j = 0; j < SIZE_OCEAN; j++) {
+                EmptySea emptySea = new EmptySea();
+                emptySea.placeShipAt(i, j, true, this);
+            }
+        }
+    }
+
+    /**
+     * Place all randomly on the (initially empty) ocean.
+     * Place larger ships before smaller ones, or you may end up with no legal place to put a large ship.
+     */
+    public void placeAllShipsRandomly() {
+        Ship[] ships =  newShips();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "ships" which hides the field declared at line 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)